packages/playground/vite.config.ts

```diff
@@ -2,10 +2,30 @@ import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 import path from 'node:path';
 import { fileURLToPath } from 'url';
+import { hmrPlugin, presets } from 'vite-plugin-web-components-hmr';
+
+const customLitPath = path.resolve(
+  __dirname,
+  '../blocks/src/__internal__/index.js'
+);
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()],
+  plugins: [
+    react(),
+    hmrPlugin({
+      include: ['../blocks/src/**/*.ts'],
```

Comment:
use absolute path?

Reply:
Not sure whether the abs path works, needs to take a look at the source code. But using a relative path is documented in the doc.